app/javascript/src/Dashboard/components/ApiFilter.jsx

```diff
@@ -1,5 +1,5 @@
-// TODO: @flow
-// TODO: test
```

Comment:
Testing is still a Todo

Reply:
But there are tests already https://github.com/3scale/porta/blob/master/spec/javascripts/Dashboard/ApiFilter.spec.jsx

Reply:
Ahhhh ;)